src/mpid/ch4/generic/am/mpidig_am_part.h

```diff
@@ -129,7 +128,7 @@ MPL_STATIC_INLINE_PREFIX int MPIDIG_mpi_parrived(MPIR_Request * request, int par
         *flag = FALSE;
 
         /* Trigger progress to process AM packages in case wait with parrived in a loop. */
-        mpi_errno = MPID_Progress_test(NULL);
+        mpi_errno = MPIDI_progress_test_vci(0);
```

Comment:
This call will eventually do global progress, right? That is my understanding from reading the code, just want to confirm.

Reply:
Yes. 

As a separate topic, I was looking at this issue https://github.com/pmodels/mpich/issues/1102, and I think adding a global progress CVAR would be a good idea. We can call e.g. `MPID_check_global_progress` in calls that normally wouldn't invoke device progress, e.g. `MPI_Send`, with the CVAR denote the polling frequency. This could mitigate Forum's concern that whether some calls should guarantee progress.